mysql-async/src/test/java/com/github/jasync/sql/db/mysql/ContainerHelper.java

```diff
@@ -1,13 +1,13 @@
 package com.github.jasync.sql.db.mysql;
 
-import com.github.jasync.sql.db.Configuration;
-import com.github.jasync.sql.db.Connection;
-import com.github.jasync.sql.db.QueryResult;
-import com.github.jasync.sql.db.RowData;
+import com.github.jasync.sql.db.*;
```

Comment:
please don't add star imports

Reply:
Sorry IntelliJ does this by default. Reverted.